app/views/components/Warning.scala.html

```diff
@@ -0,0 +1,24 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(content: Html, classes: String = "govuk-warning-text__text", id: Option[String] = None)
```

Comment:
Nice, shall we default an id here? Something like `warning-text` or `notice`?

Reply:
https://github.com/hmrc/advance-valuation-rulings-frontend/pull/17/commits/3201e0e4b93db1ee3c25c4e0415dabb5c647344b